src/components/Project/ImageAnalysis/ImageViewer.vue

```diff
@@ -19,84 +20,92 @@ const props = defineProps({
 
 const emit = defineEmits(['analysisAction'])
 
-const isLoading = ref(true)
-
-const imageContainer = ref(null)
-let image = null
+let imageMap = null
 let imageOverlay = null
 let imageBounds
 let initialCenter = [0, 0]
 let initialZoom = 1
-let lastDrawnLine = null
+let lineLayer = null
 let layerControl = null
 const imageWidth = ref(0)
 const imageHeight = ref(0)
+const isLoading = ref(true)
+const imageContainer = ref(null)
+const alerts = useAlertsStore()
 
-const scaledHeight = computed(() => imageHeight.value * 0.7)
-const scaledWidth = computed(() => imageWidth.value * 0.7)
+onMounted(() => {
+  loadImageOverlay()
+  leafletSetup()
+  resizeMap()
+})
+
+watch(() => props.catalog, () => {
+  createCatalogLayer()
+})
 
 // loads image overlay and set bounds
-const loadImageOverlay = () => {
+function loadImageOverlay() {
   // Create leaflet map (here referred to as image)
-  image = L.map(imageContainer.value, {
+  imageMap = L.map(imageContainer.value, {
     center: [0, 0],
-    zoom: 1,
+    maxZoom: 6,
     crs: L.CRS.Simple,
-    minZoom: 0,
-    dragging: false,
+    attributionControl: false,
+    maxBoundsViscosity: 1.0,
```

Comment:
value of 1.0 disables dragging past the bounds, no need for the dragging zoom check function